client/src/components/userButton/userButton.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
```

Comment:
Why is this called a userButton? Looks more like a menu to me

Reply:
It was more like a userButton before. Now it's more of a menu

Reply:
Renamed to userMenu